--- linux-custom.org/arch/arm/boot/dts/bcm2837.dtsi	2020-02-17 14:26:50.373341746 +0100
+++ linux-custom/arch/arm/boot/dts/bcm2837.dtsi	2020-02-17 14:27:23.173348537 +0100
@@ -16,6 +16,18 @@
 			#interrupt-cells = <2>;
 			interrupt-parent = <&local_intc>;
 		};
+		
+		
+		firmware {
+			optee {
+				compatible = "linaro,optee-tz";
+				method = "smc";
+			};
+			psci {
+				compatible = "arm,psci-1.0", "arm,psci-0.2";
+				method = "smc";
+			};
+		};
 	};
 
 	arm-pmu {
@@ -37,38 +49,33 @@
 	cpus: cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		enable-method = "brcm,bcm2836-smp"; // for ARM 32-bit
 
 		cpu0: cpu@0 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
 			reg = <0>;
-			enable-method = "spin-table";
-			cpu-release-addr = <0x0 0x000000d8>;
+			enable-method = "psci";
 		};
 
 		cpu1: cpu@1 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
 			reg = <1>;
-			enable-method = "spin-table";
-			cpu-release-addr = <0x0 0x000000e0>;
+			enable-method = "psci";
 		};
 
 		cpu2: cpu@2 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
 			reg = <2>;
-			enable-method = "spin-table";
-			cpu-release-addr = <0x0 0x000000e8>;
+			enable-method = "psci";
 		};
 
 		cpu3: cpu@3 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a53";
 			reg = <3>;
-			enable-method = "spin-table";
-			cpu-release-addr = <0x0 0x000000f0>;
+			enable-method = "psci";
 		};
 	};
 };
